### DISCUSSION TITLE
How I can set reviewer to pr

### DISCUSSION THREAD

**Discussion by underhooddev** [community/community#70081] [Pull Requests]
### Select Topic Area

Question

### Body

I want to set reviewer to to my pr how I can.

**Comment by PriyanshuPz** [comment]
You can set  reviewer your pr by going to top right clicking to assign setting and selecting user.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
